projects/Capture_Video_Frames/capture_vido_from_camera.py

```diff
@@ -0,0 +1,23 @@
+import cv2
+
+# cv2 method to capture frames from live video
+cap = cv2.VideoCapture(0)
```

Comment:
is it possible to handle multiple webcams and selecting appropriate one?

Reply:
Yes it could be possible if you have multiple cameras attached to the system